packages/maskbook/src/extension/options-page/DashboardComponents/WalletAssetsTable.tsx

```diff
@@ -174,8 +174,10 @@ function ViewDetailed(props: ViewDetailedProps) {
                         </TableCell>
                     ))}
             </TableRow>
-            <div className={classes.menuAnchorElRef}></div>
-            {menu}
+            <tr className={classes.menuAnchorElRef}></tr>
```

Comment:
Warning: validateDOMNesting(...): <div> cannot appear as a child of <tbody>.

Reply:
```suggestion
            <tr className={classes.menuAnchorElRef} />
```

Reply:
Why not use `<TableRow />`?

Reply:
It is not a row actually, it is just a ref for menu popup position.